Expected stderr:

✔ connect X.Xs
│ ✔ starting engine X.Xs
│ ✔ connecting to engine X.Xs
│ ✔ starting session X.Xs

✔ loading module X.Xs
│ ✔ finding module configuration X.Xs
│ ✔ serving module X.Xs
│ ✔ inspecting module X.Xs

✔ parsing command line arguments X.Xs

✔ python: Python! X.Xs
✘ .pending: Void X.Xs
! process "false" did not complete successfully: exit code: 1
│ $ Container.from(address: "alpine:latest"): Container! X.Xs CACHED
│ │ ✔ resolving docker.io/library/alpine:latest X.Xs
│ ✔ .withEnvVariable(name: "NOW", value: "20XX-XX-XX XX:XX:XX.XXXX"): Container! X.Xs
│ ✔ .withExec(args: ["sleep", "1"]): Container! X.Xs
│ ✘ .withExec(args: ["false"]): Container! X.Xs
│ ! process "false" did not complete successfully: exit code: 1
│ ○ .withExec(args: ["sleep", "1"]): Container! X.Xs

Setup tracing at https://dagger.cloud/traces/setup. To hide: export NOTHANKS=1
